pyhelm3/models.py

```diff
@@ -43,7 +43,7 @@ def __init__(self, _command: Command, **kwargs):
 
 
 #: Type for a name (chart or release)
-Name = constr(pattern = r"^[a-z0-9-]+$")
+Name = constr(pattern = r"^[a-z0-9_-]+$")
```

Comment:
I think its because of the Chart name restrictions here:
https://helm.sh/docs/chart_best_practices/conventions/#chart-names

Maybe we need a different type for the release names, which are not as strict, it would seem?